packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCard/AddCard.types.ts

```diff
@@ -0,0 +1,61 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+
+export interface IAddCardProps {
+  id: string;
+  /**
+   * The title for card representation in add card panel
+   */
+  title: string;
+
+  /**
+   * The image url for card representation in add card panel
+   */
+  imageSrc?: string;
+
+  /**
+   * The description for card representation in add card panel
+   */
+  description?: string;
+
+  /**
+   * The description for card representation in add card panel
```

Comment:
comment?

Reply:
This interface is not used by the user, only used internally by code. The same thing with above comment on id.